.github/workflows/codecov.yml

```diff
@@ -24,6 +24,11 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
 
+      - name: Install node
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20
```

Comment:
What's this for?

Reply:
In `_test.mjs` I used `import.meta.dirname` which requires node 20